datalad/distributed/create_sibling_gitlab.py

```diff
@@ -395,32 +407,33 @@ def _proc_dataset(refds, ds, site, project, remotename, layout, existing,
             "Unknown site access '{}' given or configured, "
             "known ones are: {}".format(access, known_access_labels))
 
+    pathsep = ds.config.get("datalad.gitlab-default-pathseparator", None) or "-"
```

Comment:
```suggestion
    pathsep_config = "datalad.gitlab-default-pathseparator"
    pathsep = ds.config.get(pathsep_config,
                            refds.config.get(pathsep_config, "-"))
```

Reply:
This checks the dataset configuration, then the top-level dataset config, and lastly takes the "-" as default.

Reply:
The same probably has to be done for the `project_stub`?